app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -11,18 +12,24 @@ import { getTokenIcon } from '../../../../components/icons/tokens/_index'
 import { useWhaleApiClient } from '../../../../contexts/WhaleContext'
 import { fetchTokens, useTokensAPI } from '../../../../hooks/wallet/TokensAPI'
 import { RootState } from '../../../../store'
+import { ocean } from '../../../../store/ocean'
 import { WalletToken } from '../../../../store/wallet'
 import { translate } from '../../../../translations'
 import { BalanceParamList } from './BalancesNavigator'
 
 type Props = StackScreenProps<BalanceParamList, 'BalancesScreen'>
 
 export function BalancesScreen ({ navigation }: Props): JSX.Element {
+  const height = useBottomTabBarHeight()
   const client = useWhaleApiClient()
   const address = useSelector((state: RootState) => state.wallet.address)
   const [refreshing, setRefreshing] = useState(false)
   const dispatch = useDispatch()
 
+  useEffect(() => {
+    dispatch(ocean.actions.setHeight(height))
+  }, [height])
```

Comment:
slightly uncomfortable with this height injection, hope we can refactor in the future

Reply:
yeah, I'll check if we can have static numbers for it or if we have a better way to get exact height. I'll create a ticket for it